erpnext/stock/doctype/pick_list/pick_list.json

```diff
@@ -126,11 +130,32 @@
    "fieldtype": "Check",
    "label": "Group Same Items",
    "print_hide": 1
+  },
+  {
+   "fieldname": "column_break_13",
+   "fieldtype": "Column Break"
+  },
+  {
+   "default": "0",
+   "fieldname": "scan_mode",
```

Comment:
I don't think this is required. We can always show the scan_barcode field (which has global setting (in stock settings) to hide it if required 😄 


This was required in stock reconciliation because there the default behaviour and scan behaviour were quite conflicting in nature. Here there are no triggers on picked qty.